flowr/src/main/java/com/fueled/flowr/Flowr.java

```diff
@@ -251,6 +288,46 @@ protected final String getTagPrefix() {
     }
 
     /**
+     * Parse the intent set by {@link #setDeepLinkContext(Intent)} and if this intent contains Deep Link info, update the {@link #currentFragment} and the Transaction data.
+     *
+     * @param data The Transaction data to extend if Deep link info are found in the {@link #deepLinkIntent}.
+     * @param <T>  The generic type for a valid Fragment.
+     */
+    @SuppressWarnings("unchecked")
+    private <T extends Fragment & FlowrFragment> void injectDeepLinkInfo(TransactionData<T> data) {
+        if (deepLinkIntent != null) {
+            try {
+                if (deepLinkHandler == null) {
```

Comment:
@julien-fueled maybe we can move this inside `findBindingConstructorForClass()` and rename the method to `getDeepLinkHandler()` so we would be able to call `getDeepLinkHandler()` anywhere we want access to it.